src/main/java/net/conjur/api/Token.java

```diff
@@ -91,7 +123,12 @@ public static Token fromJson(String json){
         return new Token(json);
     }
 
-	public String toBase64(){
+    private String fromBase64(String base64){
+        // NB url safe mode *does not* work
+        return new String(Base64.decodeBase64(base64), StandardCharsets.UTF_8);
+    }
+
+    public String toBase64(){
         // NB url safe mode *does not* work
```

Comment:
what specifically doesn't work? is this something we should document or fix?

Reply:
My bad.
It is a matter of copy/paste from the toBase64 method.
I have removed the comment.